ballerina/response_formatter.bal

```diff
@@ -178,6 +184,15 @@ class ResponseFormatter {
             return <__Field>getFieldFromFieldArray(fields, fieldName);
         }
     }
+
+    isolated function getFieldValue(Data data, parser:FieldNode fieldNode) returns anydata|anydata[] {
+        if data.hasKey(fieldNode.getAlias()) {
+            return data[fieldNode.getAlias()];
+        } else if data.hasKey(fieldNode.getName()) {
```

Comment:
Do we need the `else if` block here? If `data.hasKey(fieldNode.getAlias())` fails, it must have `fieldNode.getName()` key right?